slsim/lens.py

```diff
@@ -688,14 +767,16 @@ def point_source_magnification(self):
                 )
         return self._ps_magnification_list
 
-    def _point_source_magnification(self, source):
+    def _point_source_magnification(self, source, extended=False):
         """Macro-model magnification of a point source. This is for a single
-        source.
+        source. The function also works for extended source.
```

Comment:
Be more explicit: state that it will use the center of the extended source instead to calculate the lensing magnifications